geometry/eolearn/geometry/transformations.py

```diff
@@ -10,7 +10,7 @@
 file in the root directory of this source tree.
 """
 
-import functools
+from functools import partial
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#imports) recommends `import functools` and then calling `functools.partial`. That is because the name `partial` by itself can easily be mistaken for a variable or some custom function.

Reply:
but then i have to break the code up in 3 lines :(
(will fix)